README.md

```diff
@@ -54,29 +78,41 @@ You can also add the following to your `containers/.env` file instead
 COMPOSE_FILE=docker-compose.yml:docker-compose.ssh-agent.yml
 ```
 
-This may only be needed on OS X.
-
-It assumes, you don't run your stuff as root. If you do, you can leave the socat stuff out.
+##### Add your ssh keys
 
-```bash
+```sh
 KEY=id_rsa
-
-docker run --rm --volumes-from=pontsun_ssh_agent -v ~/.ssh/$KEY:/.ssh/$KEY -it nardeas/ssh-agent ssh-add -l 
-if [[ $? == 1 ]]; then
-    docker run --rm --volumes-from=pontsun_ssh_agent -v ~/.ssh/$KEY:/.ssh/$KEY -it nardeas/ssh-agent ssh-add /root/.ssh/id_rsa
-fi
-
-docker-compose exec -d $YOUR_CONTAINER ./socat.sh
-docker-compose exec  $YOUR_CONTAINER sudo chown $YOUR_USER /home/$YOUR_USER/.ssh/socket
+docker run --rm --volumes-from=pontsun_sshagent -v ~/.ssh/$KEY:/root/.ssh/$KEY -it docksal/ssh-agent:latest ssh-add /root/.ssh/$KEY
```

Comment:
@Aerzas the expected container name is `pontsun_ssh_agent` instead of `pontsun_sshagent`